admin-ui/src/pages/request-detail-page.tsx

```
@@ -49,6 +49,45 @@ import {
 
 const EMPTY = "—"
 
+export function buildRequestDetailUpstreamHeaderRows(item: AdminRequestItem): Array<{
```

ESLint

[error] Replace `item:·AdminRequestItem` with `⏎··item:·AdminRequestItem,⏎`

(prettier/prettier)